### PR TITLE
misc(core): Add more details for README

### PR DESCRIPTION
**Pull request by xingyif**
**Pull Request checklist**

- [x] The commit(s) message(s) follows the contribution [guidelines](CONTRIBUTING.md) ?
- [x] ~Tests for the changes have been added (for bug fixes / features) ?~
- [x] Docs have been added / updated (for bug fixes / features) ?

**Current behavior :** 
README lacks a few small details for Kafka end-to-end testing.

**New behavior :**
Included details for running FiloDB successfully.

### CODE REVIEW

**Comment by xingyif** [comment]
Changed commit messages to follow the commit guidelines.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Contribution Guidelines
 
 Thank you for thinking of contributing to FiloDB!   We welcome all contributions through Github Pull Requests.  
+When you create a new pr, please be sure to review the guidelines below. Then create a branch based on `develop` (not `master`).
```
Comment (velvia) [inline CONTRIBUTING.md]: capital: PR

File: README.md [hunk 2]
```diff
@@ -133,11 +133,20 @@ kafka-topics --create --zookeeper localhost:2181 --replication-factor 1 --partit
 kafka-topics --create --zookeeper localhost:2181 --replication-factor 1 --partitions 4 --topic timeseries-dev-ds-1m
 ```
 
-Download and start Cassandra 2.1 or above (Cassandra 3 and above recommended).
+Download and start Cassandra 2.1 or more recent versions (Cassandra 3 and above recommended).
 
 ```
 bin/cassandra
 ```
+You should install Cassandra using a tool which you're the most familiar with. For instance, one easy way to install it is via `brew`
```
Comment (velvia) [inline README.md]: Probably want an extra blank line before this

**Comment by xingyif** [comment]
ACK. I will address your comments today after work. Thanks @velvia :)

**Comment by xingyif** [comment]
Addressed your comments. PTAL @velvia

**Comment by xingyif** [comment]
CI failed with the following error:

```
 should reset metadata correctly when recycling old write buffers *** FAILED ***
[info]   filodb.memory.OutOfOffheapMemoryException: Out of offheap memory: Need 72 but only have 19 bytes
[info]   at filodb.memory.Block.allocMetadata(Block.scala:180)
[info]   at filodb.memory.BlockMemFactory$$anonfun$endMetaSpan$1.apply(MemFactory.scala:280)
[info]   at filodb.memory.BlockMemFactory$$anonfun$endMetaSpan$1.apply(MemFactory.scala:278)
[info]   at scala.collection.immutable.List.foreach(List.scala:392)
[info]   at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
[info]   at scala.collection.mutable.ListBuffer.foreach(ListBuffer.scala:45)
[info]   at filodb.memory.BlockMemFactory.endMetaSpan(MemFactory.scala:278)
[info]   at filodb.core.memstore.TimeSeriesPartition.filodb$core$memstore$TimeSeriesPartition$$encodeOneChunkset(TimeSeriesPartition.scala:212)
[info]   at filodb.core.memstore.TimeSeriesPartition.switchBuffers(TimeSeriesPartition.scala:186)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec$$anonfun$6$$anonfun$apply$mcV$sp$2$$anonfun$apply$mcVI$sp$1.apply$mcZI$sp(TimeSeriesPartitionSpec.scala:325)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec$$anonfun$6$$anonfun$apply$mcV$sp$2$$anonfun$apply$mcVI$sp$1.apply(TimeSeriesPartitionSpec.scala:322)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec$$anonfun$6$$anonfun$apply$mcV$sp$2$$anonfun$apply$mcVI$sp$1.apply(TimeSeriesPartitionSpec.scala:322)
[info]   at scala.collection.immutable.Range.foreach(Range.scala:160)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec$$anonfun$6$$anonfun$apply$mcV$sp$2.apply$mcVI$sp(TimeSeriesPartitionSpec.scala:322)
[info]   at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:160)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec$$anonfun$6.apply$mcV$sp(TimeSeriesPartitionSpec.scala:321)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec$$anonfun$6.apply(TimeSeriesPartitionSpec.scala:292)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec$$anonfun$6.apply(TimeSeriesPartitionSpec.scala:292)
[info]   at org.scalatest.Transformer$$anonfun$apply$1.apply$mcV$sp(Transformer.scala:22)
[info]   at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   at org.scalatest.FunSpecLike$$anon$1.apply(FunSpecLike.scala:422)
[info]   at org.scalatest.Suite$class.withFixture(Suite.scala:1122)
[info]   at org.scalatest.FunSpec.withFixture(FunSpec.scala:1626)
[info]   at org.scalatest.FunSpecLike$class.invokeWithFixture$1(FunSpecLike.scala:419)
[info]   at org.scalatest.FunSpecLike$$anonfun$runTest$1.apply(FunSpecLike.scala:431)
[info]   at org.scalatest.FunSpecLike$$anonfun$runTest$1.apply(FunSpecLike.scala:431)
[info]   at org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
[info]   at org.scalatest.FunSpecLike$class.runTest(FunSpecLike.scala:431)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec.org$scalatest$BeforeAndAfter$$super$runTest(TimeSeriesPartitionSpec.scala:47)
[info]   at org.scalatest.BeforeAndAfter$class.runTest(BeforeAndAfter.scala:200)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec.runTest(TimeSeriesPartitionSpec.scala:47)
[info]   at org.scalatest.FunSpecLike$$anonfun$runTests$1.apply(FunSpecLike.scala:464)
[info]   at org.scalatest.FunSpecLike$$anonfun$runTests$1.apply(FunSpecLike.scala:464)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:413)
[info]   at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]   at scala.collection.immutable.List.foreach(List.scala:392)
[info]   at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]   at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:396)
[info]   at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:483)
[info]   at org.scalatest.FunSpecLike$class.runTests(FunSpecLike.scala:464)
[info]   at org.scalatest.FunSpec.runTests(FunSpec.scala:1626)
[info]   at org.scalatest.Suite$class.run(Suite.scala:1424)
[info]   at org.scalatest.FunSpec.org$scalatest$FunSpecLike$$super$run(FunSpec.scala:1626)
[info]   at org.scalatest.FunSpecLike$$anonfun$run$1.apply(FunSpecLike.scala:468)
[info]   at org.scalatest.FunSpecLike$$anonfun$run$1.apply(FunSpecLike.scala:468)
[info]   at org.scalatest.SuperEngine.runImpl(Engine.scala:545)
[info]   at org.scalatest.FunSpecLike$class.run(FunSpecLike.scala:468)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec.org$scalatest$BeforeAndAfter$$super$run(TimeSeriesPartitionSpec.scala:47)
[info]   at org.scalatest.BeforeAndAfter$class.run(BeforeAndAfter.scala:241)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec.org$scalatest$BeforeAndAfterAll$$super$run(TimeSeriesPartitionSpec.scala:47)
[info]   at org.scalatest.BeforeAndAfterAll$class.liftedTree1$1(BeforeAndAfterAll.scala:257)
[info]   at org.scalatest.BeforeAndAfterAll$class.run(BeforeAndAfterAll.scala:256)
[info]   at filodb.core.memstore.TimeSeriesPartitionSpec.run(TimeSeriesPartitionSpec.scala:47)
[info]   at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:357)
[info]   at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:502)
[info]   at sbt.ForkMain$Run$2.call(ForkMain.java:296)
[info]   at sbt.ForkMain$Run$2.call(ForkMain.java:286)
[info]   at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[info]   at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[info]   at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[info]   at java.base/java.lang.Thread.run(Thread.java:834)
```

Is this a flaky test?

**Comment by xingyif** [comment]
Pushed a bit more details, rebuild CI, the flaky test is gone now.

**Pull request merged**